jwst/extract_1d/extract_1d_step.py

```diff
@@ -73,7 +73,7 @@ class Extract1dStep(Step):
     apply_nod_offset = boolean(default=None)
     """
```

Comment:
I'm thinking it couldn't hurt to add an optional argument here to allow for turning the apcorr on and off, e.g. "apply_apcorr = boolean(default=True)", at least to support testing (i.e. turn it off and verify that results from before have not changed). There may also be times where a user doesn't want to apply it. Of course you also then need to add logic into the code to make use of that param.